docs/jwst/residual_fringe/arguments.rst

```diff
@@ -3,4 +3,15 @@
 Step Arguments
 ==============
 
-The residual fringe step has no step-specific arguments.
+The residual fringe step has two step arguments that can be used to ignore determining a residual fringe
+correction in a wavelength range. These two parameters must have the same number of elements. 
```

Comment:
Minor nit, but I think cleaner wording would be: "... two step arguments that can be used to specify wavelength regions in which no correction will be determined. The two arguments give lists of minimum and maximum wavelength values, respectively, for the regions to be ignored. The two lists must contain an equal number of elements."